arai60/11-16_HashMap/14_929_Unique Email Addresses/level_3.py

```diff
@@ -0,0 +1,8 @@
+class Solution:
+    def numUniqueEmails(self, emails: List[str]) -> int:
+        receive_mails = set()
+        for email in emails:
+            local_part, domain = email.rsplit("@", 1)
+            local_name = local_part.split("+")[0].replace(".","")
+            receive_mails.add(local_name + "@" + domain)
```

Comment:
好みですがf-stginsを使っても良いかもですね。
https://docs.python.org/3/reference/lexical_analysis.html#formatted-string-literals

```suggestion
            receive_mails.add(f"{local_name}@{domain})
```